tests/sentry/integrations/msteams/test_integration.py

```diff
@@ -78,9 +78,9 @@ def assert_setup_flow(self):
                 integration=integration, organization=self.organization
             )
 
-            integration_url = "organizations/{}/rules/".format(self.organization.slug)
-            assert integration_url in responses.calls[1].request.body
-            assert self.organization.name in responses.calls[1].request.body
+            integration_url = u"organizations/{}/rules/".format(self.organization.slug)
```

Comment:
Added this just because it's bad practice to use `format` on non unicode strings.